plugins/feeds/public/yaraify.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from datetime import timedelta
+from io import BytesIO
+from typing import ClassVar
+from zipfile import ZipFile
+
+import yara
+
+from core import taskmanager
+from core.schemas import indicator, task
+
+
+class Yaraify(task.FeedTask):
+    _defaults = {
+        "frequency": timedelta(days=1),
+        "name": "Yaraify",
+        "description": "This feed contains yara rules",
+        "source": "",
+    }
+
+    _SOURCE_ALL_RULES: ClassVar["str"] = (
+        "https://yaraify.abuse.ch/yarahub/yaraify-rules.zip"
+    )
+
+    def run(self):
+        response = self._make_request(self._SOURCE_ALL_RULES)
+        if not response:
+            return
+        zip_file = BytesIO(response.content)
+        with ZipFile(zip_file) as zfile:
+            for name in zfile.namelist():
+                if name.endswith(".yar"):
+                    self.analyze_entry(zfile.read(name).decode("utf-8"))
+
+    def analyze_entry(self, entry: str):
+        logging.debug(f"Yaraify: {entry}")
+        try:
+            yara_rules = yara.compile(source=entry)
+        except yara.SyntaxError as e:
+            logging.error(f"Error compiling yara rule: {e}")
+            return
+        for r in yara_rules:
+            ind_obj = indicator.Indicator(
+                name=f"{r.identifier}",
+                pattern=entry,
+                type=indicator.IndicatorType.yara,
+                diamond=indicator.DiamondModel.capability,
+            )
+
+            ind_obj.description = f"{r.meta.get('description', 'N/A')}"
```

Comment:
you can pass in description in the constructor above. and no need to use the format string
```suggestion
            ind_obj.description = r.meta.get('description', 'N/A')
```